roles/ipxe/tasks/main.yml

```diff
@@ -12,6 +12,11 @@
   apt: name=genisoimage state=present
   sudo: yes
 
+- name: install binutils-dev libiberty-dev
+  apt: name=binutils-dev state=present
+  apt: name=libiberty-dev state=present
```

Comment:
Can you do this with ansible, two `apt` objects in the same task? Either way, probably better written as:

```
- name: install binutils-dev and libiberty-dev
  apt: name={{ item }} state=present
  sudo: yes
  with_items:
    - binutils-dev
    - libiberty-dev
```